cmd/dep/root_analyzer.go

```diff
@@ -124,17 +127,32 @@ func (a *rootAnalyzer) DeriveManifestAndLock(dir string, pr gps.ProjectRoot) (gp
 }
 
 func (a *rootAnalyzer) FinalizeRootManifestAndLock(m *dep.Manifest, l *dep.Lock) {
-	// Remove dependencies from the manifest that aren't used
-	for pr := range m.Constraints {
-		var used bool
-		for _, y := range l.Projects() {
-			if pr == y.Ident().ProjectRoot {
-				used = true
-				break
+	if a.gopath {
```

Comment:
We need some way to make root_analyzer aware of `-gopath` flag. If else part of this is kept here itself, in root_analyzer, it would execute the feedback code. So, in gopath mode, feedbacks would be logged twice.